docker/Dockerfile.public

```
@@ -0,0 +1,19 @@
+FROM node:24-alpine
+LABEL source_repository="https://github.com/cobaltcore-dev/aurora-dashboard"
+
+RUN apk upgrade --no-cache --no-progress && apk add --no-cache ca-certificates curl
+RUN set -e; curl -fL http://aia.pki.co.sap.com/aia/SAP%20Global%20Root%20CA.crt | tr -d '\r' > /usr/local/share/ca-certificates/SAP_Global_Root_CA.crt \
+  && update-ca-certificates;
+
+WORKDIR /app
+ADD . .
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)